src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedMethodSymbol.cs

```diff
@@ -186,6 +186,14 @@ public override bool IsImplicitlyDeclared
             }
         }
 
+        internal override bool IsInterceptable
```

Comment:
> internal override bool IsInterceptable [](http://example.com/codeflow?start=7&length=39)

I do not think it is appropriate to override this method here. Leaves should implement a method like that.

Reply:
OK, should we also go back at some point and reconsider implementations of members like `HasUnscopedRefAttribute` in this type?

https://github.com/dotnet/roslyn/blob/8b694c1bf2e94274b0e68d5a6e1cb77dddf119e7/src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedMethodSymbol.cs#L372

It feels like a similar situation--checking "does the method have this well-known attribute".

Reply:
Have opened #67580.